analyses/molecular-subtyping-embryonal/run-embryonal-subtyping.sh

```diff
@@ -8,10 +8,17 @@
 set -e
 set -o pipefail
 
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
 # This option controls whether on not the step that generates the subset
 # files gets run -- it will be turned off in CI
 SUBSET=${OPENPBTA_SUBSET:-1}
 
+# Generate JSON file with strings for inclusion/exclusion criteria 
+Rscript --vanilla 00-embryonal-select-pathology-dx.R
+ pwd
```

Comment:
```suggestion
 pwd
```
i think this was left in by mistake?